tests/Service/SnippetGeneratorTest.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace App\Tests\Controller;
+
+use PHPUnit\Framework\TestCase;
+
+class SnippetGeneratorTest extends TestCase
+{
+}
```

Comment:
no tests for SnippetGenerator service? 😢

Reply:
I know that @magobaol was working on `SnippetGeneratorTest`